tests/console/python3_new_version_check.pm

```diff
@@ -22,22 +22,20 @@ use registration "add_suseconnect_product";
 
 sub run {
     select_serial_terminal;
-    if (is_sle('>=15-SP4')) {
-        my $python_version = script_output("rpm -q python3 | awk -F \'-\' \'{print \$2}\'");
-        if ((package_version_cmp($python_version, "3.6") < 0) ||
-            (package_version_cmp($python_version, "3.7") >= 0)) {
-            # Factory default Python3 version for SLE15-SP4+ should be 3.6
-            die("Python default version differs from 3.6");
-        }
-        add_suseconnect_product('sle-module-python3');
+    # Test system python3 version
+    my $system_python_version = script_output(qq[zypper se --installed-only --provides '/usr/bin/python3' | awk -F '|' '/python3[0-9]*/ {gsub(" ", ""); print \$2}' | awk -F '-' '{print \$1}' | uniq]);
+    record_info("Installed python version: $system_python_version");
+    assert_script_run("[ -f man_or_boy.py ] || curl -O " . data_url("console/man_or_boy.py") . " || true");
+    my $sub_version = substr($system_python_version, 7);
+    my $man_or_boy = $sub_version ? script_output("python3.$sub_version man_or_boy.py") : script_output("$system_python_version man_or_boy.py");
+    if ($man_or_boy != -67) {
+        die("Execution of 'man_or_boy.py' with $system_python_version is not correct\n");
     }
```

Comment:
Can we simply use ` /usr/bin/python3 man_or_boy.py` to run the python script? because it is a softlink to default python3 binary.

# ll /usr/bin/python3
lrwxrwxrwx 1 root root 9 Jul  6  2022 /usr/bin/python3 -> python3.6